pecos/xmc/xlinear/model.py

```diff
@@ -427,6 +427,7 @@ def predict(
         X,
         pred_params=None,
         selected_outputs_csr=None,
+        batch_size=None,
```

Comment:
I would suggest to include `batch_size` into `kwargs`.

Reply:
Thanks for the suggestion. I will change it in the amendment.